addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5433,7 +5442,7 @@ msgstr ""
 
 #: xbmc\osx\XBMCHelper.cpp
 msgctxt "#13145"
-msgid "If you are currently using the Apple Remote to control"
+msgid "If you are using the Apple Remote to control"
```

Comment:
Isnt this totally redundant? You either using or not using something, grammatically its unnecessary and incorrect.